src/pip/_internal/commands/install.py

```diff
@@ -74,6 +74,10 @@ def build_wheels(
     builder,              # type: WheelBuilder
     pep517_requirements,  # type: List[InstallRequirement]
     legacy_requirements,  # type: List[InstallRequirement]
+    wheel_cache,           # type: WheelCache
```

Comment:
nit: type annotations aren't aligned anymore

Reply:
Yes, on purpose to reduce diff noise, and because black will disalign them anyway ;)